test/netlib/http/test_response.py

```diff
@@ -43,7 +43,7 @@ class TestResponseUtils(object):
     def test_get_cookies_none(self):
         resp = tresp()
         resp.headers = Headers()
-        assert not resp.cookies
+        assert len(resp.cookies) == 0
```

Comment:
I'd actually say that `not resp.cookies` looks *a bit*  more pythonic, but I don't have a strong opinion on this. :relaxed: